.github/CODEOWNERS

```diff
@@ -0,0 +1,53 @@
+# CODEOWNERS info & syntax
+# Lines starting with "#" are comments.
+#
+# Each line is a file pattern followed by one or more owners.
+#
+# Order is important; the last matching pattern takes the most
+# precedence.
+#
+# Owners can be specified by email address or GitHub username
+#
+# Teams can be specified as code owners as well. Teams should
+# be identified in the format @org/team-name. Teams must have
+# explicit write access to the repository.
+#
+# Patterns
+#
+# Whole repository
+# * @global-owner
+#
+# Directory (without subdirectories)
+# docs/* @tech_writer
+#
+# Directory (including subdirectories)
+# apps/ @app_developer
+#
+# Adding a leading "/" to a pattern means the directory must
+# be in the root of the repository.
+#
+# Empty Pattern -> no owner (@app_developer owns all of apps/ except apps/github)
+# apps/ @app_developer
+# apps/github
+
+# Uyuni Code Owners
+
+# Release Engineering
+rel-eng/ @release-engineering
+tito.props @release-engineering
+
+# Cobbler
+java/conf/cobbler/snippets/ @SchoolGuy
+
+# Python
+*.py @uyuni-project/python
+backend/satellite_exporter/ @uyuni-project/python @lucidd
+
+# Frontend
+web/ @uyuni-project/frontend
+branding/ @uyuni-project/frontend
+susemanager-frontend/ @uyuni-project/frontend
+*.jsp @uyuni-project/frontend
+
+# Testsuite
+testsuite/ @uyuni-project/qe
```

Comment:
I can't actually find where the groups are defined on Github (perhaps you could link me?), but I think it would be nice if this was q**a** similar to how the team is named on Confluence, the channel is named on Slack etc.

Reply:
You can find them on https://github.com/orgs/uyuni-project/teams/developers/teams (or on https://github.com/orgs/uyuni-project/teams if you click on the dropdown symbol at the right edge of the table row)

I wanted to not re-use the SUSE specific squad names but rather use names that can be understood by community contributors (in case they e.g. want to ping someone with "frontend" knowledge). In the case of QA/QE I asked the squad they got the answer they'd rather be named after their internal name (which changed from QA to QE to add more confusion). 

Writing this out, I realize that QE is directly going against the goal I just described because it is not intuitive. If you're fine with being called "Uyuni QA", I think it would help if we change the team name @uyuni-project/qe.

Reply:
I don't have a very strong preference one way or another, I think QA themselves might be the best to decide how to call themselves. I think your motivation makes a lot of sense.